### PR TITLE
chore(release): 0.3.1

### PR DESCRIPTION
**Pull request by marioferh**

### CODE REVIEW

**Review by marioferh — COMMENTED** [review]
ready after this merge: https://github.com/rhobs/observability-operator/pull/510
@simonpasquier

**Comment by simonpasquier** [comment]
/hold
can you rebase?

**Comment by marioferh** [comment]
> /hold can you rebase?

done

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### [0.3.1](https://github.com/rhobs/monitoring-stack-operator/commit/) (2024-06-12)
+
+### [0.3.1](https://github.com/rhobs/monitoring-stack-operator/commit/) (2024-06-11)
```
Comment (simonpasquier) [inline CHANGELOG.md:7]: hmm not sure why we have 2 entries?

**Review by simonpasquier — APPROVED** [review]
/lgtm
/approve

**Pull request merged**